src/OmniSharp.Abstractions/Models/v1/TypeLookup/DocumentationComment.cs

```diff
@@ -147,7 +147,7 @@ public static DocumentationComment From(string xmlDocumentation, string lineEndi
         private static string TrimMultiLineString(string input, string lineEnding)
         {
             var lines = input.Split(new string[] { "\n", "\r\n" }, StringSplitOptions.RemoveEmptyEntries);
-            return string.Join(lineEnding, lines.Select(l => l.TrimStart()));
+            return string.Join(lineEnding, lines.Select(l => RetainSingleLeadingSpace(l)));
```

Comment:
What happens if you put `  ` (space space) in your list of strings to split on? I think that will give you strings that start with 0 or 1 spaces...

Reply:
(Just spitballing, you don't *have* to do this)

Reply:
I dont think that will work. Suppose the user separates two works with 3 spaces "hello     hello1" , then these parts will also be split and joined with a `lineEnding` . That will be unnecessary splitting and incorrect joining.